_events/2018-12-14-galaxy-training-rostock.md

```diff
@@ -0,0 +1,48 @@
+---
+site: main
+tags: [training]
+title: Galaxy for linking Bisulfite sequencing with RNA sequencing 06.-08.03.2019 in Rostock
+starts: 2019-03-06
+ends: 2019-03-08
+organiser:
+  name: de.STAIR
+  email: destair@leibniz-fli.de
+location:
+  name: University of Rostock
+  street: Albert-Einstein-Str. 22
+  postal: 18059
+  city: Rostock
+  region: Mecklenburg-Vorpommern
+  country: Germany
+supporters:
+- denbi
+---
+
+Galaxy for linking Bisulfite sequencing with RNA sequencing, March 2019, registration is open!
+
+Contents:
+The training course will introduce the Galaxy framework (https://usegalaxy.org), and showcase its tools for data handling and preprocessing, differential gene expression analysis, bisulfite sequencing analysis, and Oxford Nanopore data exploration and usage. Lectures will be provided on the topics of RNA and Bisulfite sequencing, Epigenetics, and basic statistics, and followed by hands-on sessions. Finally, we will showcase a Galaxy Docker installation for customized workflow analysis.
+
+Lectures and hands-on sessions will focus on the following topics:
+- Next Generation Sequencing
+- Epigenetics
+- Workflow development
+- Differential gene expression analysis
+- Differential methylation Analysis
+- Oxford Nanopore data processing
+
+Learning goals:
+- Know your data, and the implications of data preprocessing and quality control
+- Performing differential gene expression analysis using Galaxy
+- Performing differential methylation analysis using Galaxy
+- Handling and analyze Oxford Nanopore sequencing data
+
+Prerequisites:
+This workshop is intended for MSc/PhD students, postdocs, and researchers with a background in life science research. During registration, please hand over a poster about a research topic of your choosing. Posters will be presented during our poster session, that you can use to exchange experiences with other researchers.
+
+Keywords:
+Galaxy, Workflow development, Differential gene expression analysis, differential methylation analysis, Epigenetics, Oxford Nanopore
+
+For program and registration, please visit the de.NBI Program [page](https://www.denbi.de/training/532-galaxy-for-linking-bisulfite-sequencing-with-rna-sequencing).
```

Comment:
I would add the direct link for registration: https://www.leibniz-fli.de/de/news-events/seminare/news-detailpage/news/denbidestair-training-course-galaxy-for-linking-bisulfite-sequencing-with-rna-sequencing/

Reply:
Will do!